src/components/select/index.stories.mdx

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
 import { action } from '@storybook/addon-actions';
 
-import { Box } from '@chakra-ui/react';
+import { Box, Spacer } from '@chakra-ui/react';
```

Comment:
Not sure where you are using the Spacer here.

Would you mind importing the Box from our index? 🙏🏽
Thank you!

Reply:
Hmm not sure how that slipped in 😁 will update the import